### DISCUSSION TITLE
OpenNept4une update

### DISCUSSION THREAD

**Discussion by Hayabusa454** [OpenNeptune3D/OpenNept4une#61] [General]
Hi there 
I am probably doing something wrong but i also know it is a simple answer when I ssh into the machine and run the opennept4une script option 1 is install/update opennept4une printer config when i do this it clears all my number in the printer.cfg is the correct or do i have something setup wrong it thought when it said update it would just do the changes you make to the configuration like the level speed change. Thanks John

**Comment by halfmanbear** [comment]
It should have backed up your old printer config.

Backup of 'printer.cfg' created as backup-printer.cfg.bak in the same directory.

I'll add clarification that this is a replacement and values need transferring from the backup.

Thankyou

**Reply by Hayabusa454** [reply]
Thank you for info, its easy enough to cut and paste my values in. It also looks like it removes some of the moonraker info, anything after mobileraker for me how is that possible it thought they were two differnt configuration? Thanks again John
